docs/block-client-apis/notices.md

```diff
@@ -0,0 +1,174 @@
+# Frontend notices
+
+## useStoreNotices()
+
+The `useStoreNotices()` hook allows reading and manipulating notices in the frontend.
+
+### API
+
+#### `addDefaultNotice( text = '', noticeProps = {} )`
+
+Create a new notice with `default` status. If the context is not specified in the `options` object, the current context is used.
+
+| Argument      | Type   | Description                                                                                                                                                                              |
+| ------------- | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `text`        | string | Text to be displayed in the notice.                                                                                                                                                      |
+| `noticeProps` | Object | Object with the notice props. Refer to the [Gutenberg docs](https://github.com/WordPress/gutenberg/blob/master/packages/notices/src/store/actions.js#L46) to know the available options. |
+
+#### `addErrorNotice( text = '', noticeProps = {} )`
+
+Create a new error notice. If the context is not specified in the `options` object, the current context is used.
```

Comment:
> If the context is not specified in the `options` object, the current context is used.

Preferably this should say `noticeProps` object, hopefully with an example, and could possibly only mentioned once instead of each section?

Reply:
Yeah, good point. Changed in: fea3e386a63271a466401d8f2bd86d7c6905aee2.